src/current/v23.2/physical-cluster-replication-technical-overview.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Physical Cluster Replication Technical Overview
+summary: Understand the technical details of CockroachDB physical cluster replication.
+toc: true
+docs_area: manage
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v23.2" %} Physical cluster replication automatically and continuously streams data from an active _primary_ CockroachDB cluster to a passive _standby_ cluster. Each cluster contains: a _system interface_ and an application _virtual cluster_:
+
+{% include {{ page.version.version }}/physical-replication/interface-virtual-cluster.md %}
+
+This separation of concerns means that the replication stream can operate without affecting work happening in the virtual cluster.
+
+### Replication stream start-up sequence
+
+Starting a physical replication stream consists of two jobs: one on the primary and one on the standby.
```

Comment:
```suggestion
Starting a physical replication stream consists of two jobs: one each on the standby and primary cluster:
```
Adjusted the order to match the list order

Reply:
Much better, thanks!